src/RawModelExportItem.tsx

```diff
@@ -0,0 +1,91 @@
+import Stack from '@mui/material/Stack'
+import Typography from '@mui/material/Typography'
+import Box from '@mui/system/Box'
+import Button from '@mui/material/Button'
+import { useState } from 'react'
+import DialogTitle from '@mui/material/DialogTitle'
+import DialogContent from '@mui/material/DialogContent'
+import Dialog from '@mui/material/Dialog'
+import { Divider } from '@mui/material'
+import DirectoryTreeView from './DirectoryTreeView'
+import { ModelUploadType } from '../types/interfaces'
+import DisabledElementTooltip from './common/DisabledElementTooltip'
+
+function RawModelExportItem({
+  deploymentUuid,
+  version,
+  uploadType,
+}: {
+  deploymentUuid: string
+  version: string
+  uploadType: string
+}) {
+  const [displayTree, setDisplayTree] = useState(false)
+
+  let primaryButtonDisabledReason: string | undefined = 'No files to download'
+  if (uploadType === ModelUploadType.Zip || uploadType === undefined) {
```

Comment:
`uploadType === undefined` is either an unnecessary check or we should update the prop definition to specify that this is an optional field.

If we go with the latter, we should always treat it as a `string` and default it to `''`. That would make then make this check `if (!uploadType || uploadType === ModelUploadType.Zip)`

Reply:
I've gone with making it optional. I believe I put this in (or did this somewhere else?) with the thought that existing / older models might not have this functionality as BuildOptions is relatively new.

Reply:
Good thinking 👍